lib/api-helper/listener/executeAutoInspects.ts

```diff
@@ -73,11 +73,11 @@ export function executeAutoInspects(autoInspectRegistrations: Array<AutoInspectR
 
 // tslint:disable
 /**
- * each inspection can return a result, which may be turned into a PushImpactResponse by its onInspectionResult,
+ * Each inspection can return a result, which may be turned into a PushImpactResponse by its onInspectionResult,
  * OR it may return a ProjectReview, which will be processed by each ProjectReviewListener. The Listener may also return a PushImpactResponse.
  * Each of these PushReactionResponses may instruct the AutoInspect goal to fail or to require approval.
  *
- * ROD: which of these paths is deprecated? I'm guessing the ReviewListener is deprecated
```

Comment:
The ReviewListener isn't deprecated... I don't think anything is deprecated here